[2024-11-06T19:34:27.200+0000] {processor.py:186} INFO - Started process (PID=668) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:34:27.200+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:34:27.202+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:34:27.202+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:34:27.233+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:34:27.243+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.047 seconds
[2024-11-06T19:34:58.124+0000] {processor.py:186} INFO - Started process (PID=731) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:34:58.125+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:34:58.127+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:34:58.127+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:34:58.136+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:34:58.147+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.027 seconds
[2024-11-06T19:35:28.626+0000] {processor.py:186} INFO - Started process (PID=794) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:35:28.627+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:35:28.629+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:35:28.629+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:35:28.641+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:35:28.654+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.032 seconds
[2024-11-06T19:35:59.170+0000] {processor.py:186} INFO - Started process (PID=857) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:35:59.171+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:35:59.172+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:35:59.172+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:35:59.182+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:35:59.194+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.028 seconds
[2024-11-06T19:36:29.428+0000] {processor.py:186} INFO - Started process (PID=920) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:36:29.429+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:36:29.430+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:36:29.430+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:36:29.439+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:36:29.450+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.026 seconds
[2024-11-06T19:36:59.835+0000] {processor.py:186} INFO - Started process (PID=983) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:36:59.838+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:36:59.856+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:36:59.845+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:36:59.906+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:36:59.936+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.106 seconds
[2024-11-06T19:37:30.783+0000] {processor.py:186} INFO - Started process (PID=1046) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:37:30.784+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:37:30.786+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:37:30.785+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:37:30.795+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:37:30.806+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.027 seconds
[2024-11-06T19:37:49.628+0000] {processor.py:186} INFO - Started process (PID=1109) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:37:49.629+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:37:49.632+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:37:49.631+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:37:49.647+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:37:49.645+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3
    with DAG()
              ^
SyntaxError: expected ':'
[2024-11-06T19:37:49.647+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:37:49.664+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.041 seconds
[2024-11-06T19:38:20.554+0000] {processor.py:186} INFO - Started process (PID=1172) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:38:20.554+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:38:20.556+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:38:20.556+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:38:20.573+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:38:20.568+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3
    with DAG()
              ^
SyntaxError: expected ':'
[2024-11-06T19:38:20.573+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:38:20.590+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.040 seconds
[2024-11-06T19:38:51.207+0000] {processor.py:186} INFO - Started process (PID=1235) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:38:51.208+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:38:51.210+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:38:51.209+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:38:51.224+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:38:51.222+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3
    with DAG()
              ^
SyntaxError: expected ':'
[2024-11-06T19:38:51.224+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:38:51.240+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.038 seconds
[2024-11-06T19:39:21.502+0000] {processor.py:186} INFO - Started process (PID=1298) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:39:21.503+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:39:21.505+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:39:21.505+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:39:21.522+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:39:21.519+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3
    with DAG()
              ^
SyntaxError: expected ':'
[2024-11-06T19:39:21.530+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:39:21.668+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.175 seconds
[2024-11-06T19:39:48.776+0000] {processor.py:186} INFO - Started process (PID=1355) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:39:48.776+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:39:48.778+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:39:48.778+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:39:48.791+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:39:48.790+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3
    with DAG('user_processing')
                               ^
SyntaxError: expected ':'
[2024-11-06T19:39:48.792+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:39:48.806+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.035 seconds
[2024-11-06T19:40:07.790+0000] {processor.py:186} INFO - Started process (PID=1362) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:40:07.791+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:40:07.794+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:40:07.793+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:40:07.812+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:40:07.810+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3
    with DAG('user_processing', start_date=)
                                ^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-11-06T19:40:07.812+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:40:07.831+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.045 seconds
[2024-11-06T19:40:38.376+0000] {processor.py:186} INFO - Started process (PID=1425) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:40:38.377+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:40:38.380+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:40:38.379+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:40:38.394+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:40:38.392+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3
    with DAG('user_processing', start_date=)
                                ^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-11-06T19:40:38.394+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:40:38.416+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.046 seconds
[2024-11-06T19:41:08.746+0000] {processor.py:186} INFO - Started process (PID=1488) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:41:08.754+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:41:08.769+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:41:08.763+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:41:08.813+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:41:08.809+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 3
    with DAG('user_processing', start_date=)
                                ^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-11-06T19:41:08.814+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:41:09.057+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.326 seconds
[2024-11-06T19:41:30.059+0000] {processor.py:186} INFO - Started process (PID=1552) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:41:30.059+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:41:30.061+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:41:30.061+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:41:30.076+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:41:30.074+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1))
                                                                ^
SyntaxError: expected ':'
[2024-11-06T19:41:30.076+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:41:30.092+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.037 seconds
[2024-11-06T19:41:30.128+0000] {processor.py:186} INFO - Started process (PID=1553) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:41:30.128+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:41:30.131+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:41:30.130+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:41:30.144+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:41:30.143+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1))
                                                                ^
SyntaxError: expected ':'
[2024-11-06T19:41:30.145+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:41:30.161+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.038 seconds
[2024-11-06T19:41:30.193+0000] {processor.py:186} INFO - Started process (PID=1554) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:41:30.194+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:41:30.196+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:41:30.196+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:41:30.208+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:41:30.208+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1))
                                                                ^
SyntaxError: expected ':'
[2024-11-06T19:41:30.209+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:41:30.226+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.037 seconds
[2024-11-06T19:41:30.260+0000] {processor.py:186} INFO - Started process (PID=1555) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:41:30.262+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:41:30.265+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:41:30.264+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:41:30.276+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:41:30.275+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1))
                                                                ^
SyntaxError: expected ':'
[2024-11-06T19:41:30.277+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:41:30.294+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.039 seconds
[2024-11-06T19:42:00.808+0000] {processor.py:186} INFO - Started process (PID=1618) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:42:00.809+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:42:00.810+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:42:00.810+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:42:00.822+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:42:00.821+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1))
                                                                ^
SyntaxError: expected ':'
[2024-11-06T19:42:00.823+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:42:00.837+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.032 seconds
[2024-11-06T19:42:31.184+0000] {processor.py:186} INFO - Started process (PID=1681) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:42:31.185+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:42:31.187+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:42:31.187+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:42:31.201+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:42:31.199+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1))
                                                                ^
SyntaxError: expected ':'
[2024-11-06T19:42:31.201+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:42:31.218+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.040 seconds
[2024-11-06T19:43:01.661+0000] {processor.py:186} INFO - Started process (PID=1744) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:43:01.662+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:43:01.664+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:43:01.663+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:43:01.677+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:43:01.674+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1))
                                                                ^
SyntaxError: expected ':'
[2024-11-06T19:43:01.677+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:43:01.708+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.053 seconds
[2024-11-06T19:43:31.960+0000] {processor.py:186} INFO - Started process (PID=1807) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:43:31.969+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:43:31.971+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:43:31.971+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:43:31.984+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:43:31.982+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1))
                                                                ^
SyntaxError: expected ':'
[2024-11-06T19:43:31.985+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:43:32.067+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.112 seconds
[2024-11-06T19:44:02.322+0000] {processor.py:186} INFO - Started process (PID=1870) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:44:02.322+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:44:02.324+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:44:02.324+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:44:02.336+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:44:02.335+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1))
                                                                ^
SyntaxError: expected ':'
[2024-11-06T19:44:02.337+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:44:02.352+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.035 seconds
[2024-11-06T19:44:32.655+0000] {processor.py:186} INFO - Started process (PID=1933) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:44:32.656+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:44:32.657+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:44:32.657+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:44:32.671+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:44:32.669+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1))
                                                                ^
SyntaxError: expected ':'
[2024-11-06T19:44:32.671+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:44:32.686+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.035 seconds
[2024-11-06T19:45:02.943+0000] {processor.py:186} INFO - Started process (PID=1996) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:45:02.943+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:45:02.945+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:45:02.945+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:45:02.958+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:45:02.956+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1))
                                                                ^
SyntaxError: expected ':'
[2024-11-06T19:45:02.958+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:45:02.972+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.034 seconds
[2024-11-06T19:45:33.319+0000] {processor.py:186} INFO - Started process (PID=2059) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:45:33.320+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:45:33.321+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:45:33.321+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:45:33.332+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:45:33.330+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1))
                                                                ^
SyntaxError: expected ':'
[2024-11-06T19:45:33.332+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:45:33.348+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.033 seconds
[2024-11-06T19:46:03.940+0000] {processor.py:186} INFO - Started process (PID=2122) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:46:03.942+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:46:03.944+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:46:03.944+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:46:03.965+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:46:03.962+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1))
                                                                ^
SyntaxError: expected ':'
[2024-11-06T19:46:03.965+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:46:04.139+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.203 seconds
[2024-11-06T19:46:34.403+0000] {processor.py:186} INFO - Started process (PID=2185) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:46:34.403+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:46:34.405+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:46:34.405+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:46:34.418+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:46:34.416+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1))
                                                                ^
SyntaxError: expected ':'
[2024-11-06T19:46:34.418+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:46:34.432+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.033 seconds
[2024-11-06T19:47:04.772+0000] {processor.py:186} INFO - Started process (PID=2248) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:47:04.773+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:47:04.774+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:47:04.774+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:47:04.786+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:47:04.784+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1))
                                                                ^
SyntaxError: expected ':'
[2024-11-06T19:47:04.787+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:47:04.802+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.034 seconds
[2024-11-06T19:47:35.456+0000] {processor.py:186} INFO - Started process (PID=2311) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:47:35.456+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:47:35.459+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:47:35.458+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:47:35.477+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:47:35.474+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1))
                                                                ^
SyntaxError: expected ':'
[2024-11-06T19:47:35.477+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:47:35.491+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.040 seconds
[2024-11-06T19:48:17.808+0000] {processor.py:186} INFO - Started process (PID=2329) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:48:17.809+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:48:17.811+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:48:17.811+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:48:17.843+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:48:17.841+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1))
                                                                ^
SyntaxError: expected ':'
[2024-11-06T19:48:17.844+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:48:17.979+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.185 seconds
[2024-11-06T19:48:24.631+0000] {processor.py:186} INFO - Started process (PID=2385) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:48:24.632+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:48:24.633+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:48:24.633+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:48:24.647+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:48:24.645+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1), schedule_interval=)
                                                                 ^^^^^^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-11-06T19:48:24.647+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:48:24.670+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.043 seconds
[2024-11-06T19:48:43.012+0000] {processor.py:186} INFO - Started process (PID=2391) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:48:43.013+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:48:43.014+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:48:43.014+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:48:43.027+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:48:43.026+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1), schedule_interval='@daily')
                                                                                            ^
SyntaxError: expected ':'
[2024-11-06T19:48:43.027+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:48:43.042+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.034 seconds
[2024-11-06T19:49:13.631+0000] {processor.py:186} INFO - Started process (PID=2454) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:49:13.632+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:49:13.634+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:49:13.634+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:49:13.644+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:49:13.643+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1), schedule_interval='@daily')
                                                                                            ^
SyntaxError: expected ':'
[2024-11-06T19:49:13.644+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:49:13.658+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.031 seconds
[2024-11-06T19:49:44.098+0000] {processor.py:186} INFO - Started process (PID=2517) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:49:44.099+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:49:44.100+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:49:44.100+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:49:44.112+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:49:44.111+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1), schedule_interval='@daily')
                                                                                            ^
SyntaxError: expected ':'
[2024-11-06T19:49:44.112+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:49:44.128+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.034 seconds
[2024-11-06T19:50:14.490+0000] {processor.py:186} INFO - Started process (PID=2580) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:50:14.491+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:50:14.493+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:50:14.493+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:50:14.504+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:50:14.504+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1), schedule_interval='@daily')
                                                                                            ^
SyntaxError: expected ':'
[2024-11-06T19:50:14.505+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:50:14.518+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.032 seconds
[2024-11-06T19:50:44.963+0000] {processor.py:186} INFO - Started process (PID=2643) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:50:44.964+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:50:44.966+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:50:44.966+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:50:44.980+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:50:44.979+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1), schedule_interval='@daily')
                                                                                            ^
SyntaxError: expected ':'
[2024-11-06T19:50:44.981+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:50:45.162+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.203 seconds
[2024-11-06T19:51:15.283+0000] {processor.py:186} INFO - Started process (PID=2706) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:51:15.284+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:51:15.286+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:51:15.286+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:51:15.300+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:51:15.298+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 4
    with DAG('user_processing', start_date=datetime(2022, 1, 1), schedule_interval='@daily')
                                                                                            ^
SyntaxError: expected ':'
[2024-11-06T19:51:15.301+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:51:15.315+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.036 seconds
[2024-11-06T19:51:29.022+0000] {processor.py:186} INFO - Started process (PID=2769) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:51:29.023+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:51:29.025+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:51:29.025+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:51:29.043+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:51:29.042+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 8
    with DAG(dag_id='user_processing', start_date=datetime(2022, 1, 1), schedule_interval='@daily', catchup=False)
                                                                                                                  ^
SyntaxError: expected ':'
[2024-11-06T19:51:29.043+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:51:29.062+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.046 seconds
[2024-11-06T19:51:58.355+0000] {processor.py:186} INFO - Started process (PID=2821) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:51:58.356+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:51:58.359+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:51:58.359+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:51:58.372+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:51:58.370+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 8
    with DAG(dag_id='user_processing', start_date=datetime(2022, 1, 1), schedule_interval='@daily', catchup=False)
                                                                                                                  ^
SyntaxError: expected ':'
[2024-11-06T19:51:58.372+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:51:58.392+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.044 seconds
[2024-11-06T19:52:16.070+0000] {processor.py:186} INFO - Started process (PID=2833) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:52:16.071+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:52:16.073+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:52:16.073+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:52:16.084+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:52:16.083+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 10
    
    ^
IndentationError: expected an indented block after 'with' statement on line 8
[2024-11-06T19:52:16.084+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:52:16.109+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.044 seconds
[2024-11-06T19:52:22.114+0000] {processor.py:186} INFO - Started process (PID=2834) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:52:22.115+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:52:22.117+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:52:22.117+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:52:22.137+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:52:22.244+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:52:22.244+0000] {override.py:1900} INFO - Created Permission View: can delete on DAG:user_processing
[2024-11-06T19:52:22.252+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:52:22.252+0000] {override.py:1900} INFO - Created Permission View: can edit on DAG:user_processing
[2024-11-06T19:52:22.257+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:52:22.257+0000] {override.py:1900} INFO - Created Permission View: can read on DAG:user_processing
[2024-11-06T19:52:22.264+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:52:22.264+0000] {override.py:1900} INFO - Created Permission View: can delete on DAG Run:user_processing
[2024-11-06T19:52:22.269+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:52:22.269+0000] {override.py:1900} INFO - Created Permission View: can create on DAG Run:user_processing
[2024-11-06T19:52:22.274+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:52:22.274+0000] {override.py:1900} INFO - Created Permission View: can read on DAG Run:user_processing
[2024-11-06T19:52:22.279+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:52:22.279+0000] {override.py:1900} INFO - Created Permission View: menu access on DAG Run:user_processing
[2024-11-06T19:52:22.280+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:52:22.280+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T19:52:22.289+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:52:22.289+0000] {dag.py:3262} INFO - Creating ORM DAG for user_processing
[2024-11-06T19:52:22.298+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:52:22.297+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T19:52:22.312+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.202 seconds
[2024-11-06T19:52:40.796+0000] {processor.py:186} INFO - Started process (PID=2897) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:52:40.797+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:52:40.800+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:52:40.799+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:52:40.820+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:52:40.830+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:52:40.830+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T19:52:40.851+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:52:40.851+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T19:52:40.869+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.079 seconds
[2024-11-06T19:53:11.027+0000] {processor.py:186} INFO - Started process (PID=2960) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:53:11.028+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:53:11.029+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:53:11.029+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:53:11.042+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:53:11.055+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:53:11.054+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T19:53:11.072+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:53:11.072+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T19:53:11.085+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.062 seconds
[2024-11-06T19:53:41.665+0000] {processor.py:186} INFO - Started process (PID=3024) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:53:41.666+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:53:41.667+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:53:41.667+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:53:41.680+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:53:41.694+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:53:41.694+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T19:53:41.711+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:53:41.710+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T19:53:41.723+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.062 seconds
[2024-11-06T19:54:00.624+0000] {processor.py:186} INFO - Started process (PID=3056) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:54:00.625+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:54:00.626+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:54:00.626+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:54:00.662+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:54:00.682+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:54:00.682+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T19:54:00.700+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:54:00.700+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T19:54:00.716+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.098 seconds
[2024-11-06T19:54:30.995+0000] {processor.py:186} INFO - Started process (PID=3119) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:54:30.996+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:54:30.998+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:54:30.998+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:54:31.010+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:54:31.040+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:54:31.040+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T19:54:31.075+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:54:31.074+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T19:54:31.092+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.103 seconds
[2024-11-06T19:55:01.589+0000] {processor.py:186} INFO - Started process (PID=3169) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:55:01.589+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:55:01.591+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:55:01.591+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:55:01.603+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:55:01.617+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:55:01.617+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T19:55:01.643+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:55:01.642+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T19:55:01.689+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.105 seconds
[2024-11-06T19:55:32.214+0000] {processor.py:186} INFO - Started process (PID=3222) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:55:32.215+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:55:32.216+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:55:32.216+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:55:32.231+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:55:32.246+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:55:32.245+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T19:55:32.263+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:55:32.263+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T19:55:32.276+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.066 seconds
[2024-11-06T19:56:02.505+0000] {processor.py:186} INFO - Started process (PID=3279) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:56:02.506+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:56:02.509+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:56:02.508+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:56:02.526+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:56:02.540+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:56:02.540+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T19:56:02.559+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:56:02.559+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T19:56:02.576+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.075 seconds
[2024-11-06T19:56:33.661+0000] {processor.py:186} INFO - Started process (PID=3342) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:56:33.662+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:56:33.664+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:56:33.664+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:56:33.677+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:56:33.695+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:56:33.695+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T19:56:33.717+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:56:33.716+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T19:56:33.732+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.076 seconds
[2024-11-06T19:57:04.305+0000] {processor.py:186} INFO - Started process (PID=3405) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:57:04.305+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:57:04.307+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:57:04.307+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:57:04.322+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:57:04.340+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:57:04.339+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T19:57:04.362+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:57:04.361+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T19:57:04.378+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.078 seconds
[2024-11-06T19:57:34.937+0000] {processor.py:186} INFO - Started process (PID=3468) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:57:34.943+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:57:34.947+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:57:34.947+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:57:34.975+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:57:34.999+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:57:34.999+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T19:57:35.075+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:57:35.074+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T19:57:35.108+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.175 seconds
[2024-11-06T19:58:14.318+0000] {processor.py:186} INFO - Started process (PID=3513) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:58:14.319+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:58:14.321+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:58:14.321+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:58:14.333+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:58:14.345+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:58:14.345+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T19:58:14.364+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:58:14.364+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T19:58:14.379+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.067 seconds
[2024-11-06T19:58:44.622+0000] {processor.py:186} INFO - Started process (PID=3571) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:58:44.624+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:58:44.633+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:58:44.627+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:58:44.647+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:58:44.676+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:58:44.670+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T19:58:44.708+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:58:44.707+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T19:58:44.731+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.113 seconds
[2024-11-06T19:59:14.780+0000] {processor.py:186} INFO - Started process (PID=3634) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:59:14.781+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:59:14.783+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:59:14.782+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:59:14.795+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:59:14.810+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:59:14.809+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T19:59:14.828+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:59:14.827+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T19:59:14.841+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.065 seconds
[2024-11-06T19:59:45.712+0000] {processor.py:186} INFO - Started process (PID=3693) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T19:59:45.713+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T19:59:45.714+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:59:45.714+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T19:59:45.727+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T19:59:45.909+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:59:45.909+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T19:59:45.927+0000] {logging_mixin.py:190} INFO - [2024-11-06T19:59:45.927+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T19:59:45.946+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.238 seconds
[2024-11-06T20:00:16.113+0000] {processor.py:186} INFO - Started process (PID=3754) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:00:16.114+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:00:16.121+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:00:16.121+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:00:16.138+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:00:16.154+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:00:16.154+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:00:16.177+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:00:16.177+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:00:16.192+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.088 seconds
[2024-11-06T20:00:46.344+0000] {processor.py:186} INFO - Started process (PID=3814) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:00:46.345+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:00:46.346+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:00:46.346+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:00:46.365+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:00:46.383+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:00:46.382+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:00:46.402+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:00:46.402+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:00:46.415+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.075 seconds
[2024-11-06T20:01:16.514+0000] {processor.py:186} INFO - Started process (PID=3872) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:01:16.517+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:01:16.522+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:01:16.520+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:01:16.539+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:01:16.554+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:01:16.554+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:01:16.578+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:01:16.578+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:01:16.595+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.086 seconds
[2024-11-06T20:01:46.962+0000] {processor.py:186} INFO - Started process (PID=3935) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:01:46.963+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:01:46.965+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:01:46.965+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:01:46.978+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:01:46.996+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:01:46.996+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:01:47.016+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:01:47.016+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:01:47.049+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.091 seconds
[2024-11-06T20:02:12.147+0000] {processor.py:186} INFO - Started process (PID=3984) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:02:12.148+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:02:12.150+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:02:12.149+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:02:12.170+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:02:12.185+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:02:12.184+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:02:12.211+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:02:12.210+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:02:12.230+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.087 seconds
[2024-11-06T20:02:42.726+0000] {processor.py:186} INFO - Started process (PID=4046) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:02:42.726+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:02:42.729+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:02:42.728+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:02:42.752+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:02:42.767+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:02:42.767+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:02:42.787+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:02:42.786+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:02:42.801+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.079 seconds
[2024-11-06T20:03:13.082+0000] {processor.py:186} INFO - Started process (PID=4109) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:03:13.082+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:03:13.084+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:03:13.084+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:03:13.113+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:03:13.130+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:03:13.130+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:03:13.155+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:03:13.155+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:03:13.170+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.093 seconds
[2024-11-06T20:03:43.504+0000] {processor.py:186} INFO - Started process (PID=4171) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:03:43.505+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:03:43.507+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:03:43.507+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:03:43.519+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:03:43.532+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:03:43.532+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:03:43.549+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:03:43.549+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:03:43.562+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.061 seconds
[2024-11-06T20:04:13.608+0000] {processor.py:186} INFO - Started process (PID=4235) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:04:13.608+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:04:13.610+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:04:13.610+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:04:13.623+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:04:13.637+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:04:13.636+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:04:13.654+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:04:13.654+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:04:13.668+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.064 seconds
[2024-11-06T20:04:44.220+0000] {processor.py:186} INFO - Started process (PID=4298) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:04:44.221+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:04:44.223+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:04:44.223+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:04:44.236+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:04:44.249+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:04:44.249+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:04:44.267+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:04:44.266+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:04:44.279+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.063 seconds
[2024-11-06T20:05:14.842+0000] {processor.py:186} INFO - Started process (PID=4361) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:05:14.843+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:05:14.845+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:05:14.845+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:05:14.859+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:05:14.873+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:05:14.873+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:05:14.891+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:05:14.891+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:05:14.905+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.067 seconds
[2024-11-06T20:05:39.535+0000] {processor.py:186} INFO - Started process (PID=4424) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:05:39.535+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:05:39.537+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:05:39.537+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:05:39.563+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:05:39.557+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 14, in <module>
    create_table = PostgresOperator(
                   ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/postgres/operators/postgres.py", line 84, in __init__
    super().__init__(conn_id=postgres_conn_id, **kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 473, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'sql'
[2024-11-06T20:05:39.564+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:05:39.582+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.051 seconds
[2024-11-06T20:05:58.991+0000] {processor.py:186} INFO - Started process (PID=4487) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:05:58.992+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:05:58.993+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:05:58.993+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:05:59.015+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:05:59.011+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 14, in <module>
    create_table = PostgresOperator(
                   ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/postgres/operators/postgres.py", line 84, in __init__
    super().__init__(conn_id=postgres_conn_id, **kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 473, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'sql'
[2024-11-06T20:05:59.016+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:05:59.030+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.043 seconds
[2024-11-06T20:06:29.520+0000] {processor.py:186} INFO - Started process (PID=4549) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:06:29.521+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:06:29.523+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:06:29.523+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:06:29.538+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:06:29.534+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 14, in <module>
    create_table = PostgresOperator(
                   ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/postgres/operators/postgres.py", line 84, in __init__
    super().__init__(conn_id=postgres_conn_id, **kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 473, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'sql'
[2024-11-06T20:06:29.539+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:06:29.553+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.036 seconds
[2024-11-06T20:06:37.620+0000] {processor.py:186} INFO - Started process (PID=4550) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:06:37.621+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:06:37.623+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:06:37.623+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:06:37.645+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:06:37.642+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 15, in <module>
    create_table = PostgresOperator(
                   ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/postgres/operators/postgres.py", line 84, in __init__
    super().__init__(conn_id=postgres_conn_id, **kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 473, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'sql'
[2024-11-06T20:06:37.646+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:06:37.661+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.058 seconds
[2024-11-06T20:06:52.738+0000] {processor.py:186} INFO - Started process (PID=4551) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:06:52.739+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:06:52.740+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:06:52.740+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:06:52.752+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:06:52.751+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 17
    postgres_conn_id=
    ^^^^^^^^^^^^^^^^^
SyntaxError: expected argument value expression
[2024-11-06T20:06:52.753+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:06:52.768+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.034 seconds
[2024-11-06T20:07:06.207+0000] {processor.py:186} INFO - Started process (PID=4614) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:07:06.208+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:07:06.210+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:07:06.209+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:07:06.250+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:07:06.244+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 15, in <module>
    create_table = PostgresOperator(
                   ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 490, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/postgres/operators/postgres.py", line 84, in __init__
    super().__init__(conn_id=postgres_conn_id, **kwargs)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 473, in apply_defaults
    raise AirflowException(f"missing keyword argument {missing_args.pop()!r}")
airflow.exceptions.AirflowException: missing keyword argument 'sql'
[2024-11-06T20:07:06.251+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:07:06.264+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.063 seconds
[2024-11-06T20:07:32.658+0000] {processor.py:186} INFO - Started process (PID=4677) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:07:32.661+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:07:32.663+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:07:32.662+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:07:32.689+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:07:32.681+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 21
    )
    ^
SyntaxError: positional argument follows keyword argument
[2024-11-06T20:07:32.691+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:07:32.749+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.097 seconds
[2024-11-06T20:08:03.727+0000] {processor.py:186} INFO - Started process (PID=4741) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:08:03.728+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:08:03.728+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:08:03.728+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:08:03.742+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:08:03.741+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/user_processing.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 991, in exec_module
  File "<frozen importlib._bootstrap_external>", line 1129, in get_code
  File "<frozen importlib._bootstrap_external>", line 1059, in source_to_code
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/user_processing.py", line 21
    )
    ^
SyntaxError: positional argument follows keyword argument
[2024-11-06T20:08:03.742+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:08:03.760+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.038 seconds
[2024-11-06T20:08:08.782+0000] {processor.py:186} INFO - Started process (PID=4742) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:08:08.783+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:08:08.784+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:08:08.784+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:08:08.805+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:08:08.887+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:08:08.887+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:08:08.904+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:08:08.904+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:08:08.923+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.146 seconds
[2024-11-06T20:08:39.101+0000] {processor.py:186} INFO - Started process (PID=4805) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:08:39.102+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:08:39.103+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:08:39.103+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:08:39.117+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:08:39.133+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:08:39.132+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:08:39.156+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:08:39.156+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:08:39.170+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.074 seconds
[2024-11-06T20:08:52.244+0000] {processor.py:186} INFO - Started process (PID=4806) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:08:52.245+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:08:52.246+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:08:52.245+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:08:52.272+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:08:52.452+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:08:52.451+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:08:52.475+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:08:52.475+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:08:52.497+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.258 seconds
[2024-11-06T20:09:02.545+0000] {processor.py:186} INFO - Started process (PID=4838) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:09:02.546+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:09:02.547+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:09:02.547+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:09:02.575+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:09:02.592+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:09:02.592+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:09:02.617+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:09:02.617+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:09:02.644+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.098 seconds
[2024-11-06T20:09:32.973+0000] {processor.py:186} INFO - Started process (PID=4889) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:09:32.974+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:09:32.975+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:09:32.974+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:09:32.989+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:09:33.088+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:09:33.088+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:09:33.114+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:09:33.114+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:09:33.141+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.172 seconds
[2024-11-06T20:09:36.966+0000] {processor.py:186} INFO - Started process (PID=4932) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:09:36.967+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:09:36.968+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:09:36.967+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:09:36.991+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:09:37.002+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:09:37.002+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:09:37.031+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:09:37.031+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:09:37.054+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.099 seconds
[2024-11-06T20:09:57.084+0000] {processor.py:186} INFO - Started process (PID=4933) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:09:57.085+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:09:57.087+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:09:57.086+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:09:57.105+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:09:57.115+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:09:57.115+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:09:57.138+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:09:57.137+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:09:57.158+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.078 seconds
[2024-11-06T20:10:27.264+0000] {processor.py:186} INFO - Started process (PID=4996) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:10:27.266+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:10:27.267+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:10:27.266+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:10:27.290+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:10:27.429+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:10:27.428+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:10:27.446+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:10:27.445+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:10:27.494+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.234 seconds
[2024-11-06T20:10:58.373+0000] {processor.py:186} INFO - Started process (PID=5059) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:10:58.374+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:10:58.375+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:10:58.375+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:10:58.390+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:10:58.406+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:10:58.406+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:10:58.425+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:10:58.425+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:10:58.440+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.071 seconds
[2024-11-06T20:11:29.164+0000] {processor.py:186} INFO - Started process (PID=5122) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:11:29.165+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:11:29.166+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:11:29.166+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:11:29.187+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:11:29.229+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:11:29.229+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:11:29.249+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:11:29.248+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:11:29.264+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.104 seconds
[2024-11-06T20:12:00.219+0000] {processor.py:186} INFO - Started process (PID=5185) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:12:00.220+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:12:00.221+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:12:00.221+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:12:00.247+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:12:00.326+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:12:00.326+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:12:00.357+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:12:00.356+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:12:00.377+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.162 seconds
[2024-11-06T20:12:10.605+0000] {processor.py:186} INFO - Started process (PID=5248) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:12:10.606+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:12:10.607+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:12:10.607+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:12:10.627+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:12:10.642+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:12:10.642+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:12:10.661+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:12:10.661+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:12:10.678+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.077 seconds
[2024-11-06T20:12:41.965+0000] {processor.py:186} INFO - Started process (PID=5311) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:12:41.966+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:12:41.967+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:12:41.967+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:12:41.982+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:12:41.997+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:12:41.996+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:12:42.015+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:12:42.015+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:12:42.031+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.070 seconds
[2024-11-06T20:13:12.574+0000] {processor.py:186} INFO - Started process (PID=5359) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:13:12.579+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:13:12.580+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:13:12.580+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:13:12.613+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:13:12.634+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:13:12.634+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:13:12.680+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:13:12.680+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:13:12.705+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.136 seconds
[2024-11-06T20:13:46.007+0000] {processor.py:186} INFO - Started process (PID=5396) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:13:46.017+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:13:46.019+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:13:46.018+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:13:46.073+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:13:46.392+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:13:46.392+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:13:46.428+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:13:46.427+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:13:46.463+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.488 seconds
[2024-11-06T20:14:07.114+0000] {processor.py:186} INFO - Started process (PID=5437) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:14:07.116+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:14:07.119+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:14:07.117+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:14:07.143+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:14:07.196+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:14:07.195+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:14:07.253+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:14:07.253+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:14:07.309+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.200 seconds
[2024-11-06T20:14:22.694+0000] {processor.py:186} INFO - Started process (PID=5500) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:14:22.694+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:14:22.695+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:14:22.695+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:14:22.719+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:14:22.752+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:14:22.752+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:14:22.771+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:14:22.771+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:14:22.788+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.098 seconds
[2024-11-06T20:14:53.224+0000] {processor.py:186} INFO - Started process (PID=5563) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:14:53.225+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:14:53.226+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:14:53.225+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:14:53.251+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:14:53.266+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:14:53.265+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:14:53.285+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:14:53.285+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:14:53.302+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.082 seconds
[2024-11-06T20:15:23.810+0000] {processor.py:186} INFO - Started process (PID=5626) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:15:23.811+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:15:23.812+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:15:23.812+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:15:23.826+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:15:23.842+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:15:23.841+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:15:23.861+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:15:23.861+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:15:23.874+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.068 seconds
[2024-11-06T20:15:54.359+0000] {processor.py:186} INFO - Started process (PID=5688) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:15:54.360+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:15:54.361+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:15:54.360+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:15:54.374+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:15:54.388+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:15:54.388+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:15:54.406+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:15:54.406+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:15:54.419+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.064 seconds
[2024-11-06T20:16:24.742+0000] {processor.py:186} INFO - Started process (PID=5752) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:16:24.742+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:16:24.743+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:16:24.743+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:16:24.757+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:16:24.772+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:16:24.772+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:16:24.790+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:16:24.790+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:16:24.803+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.066 seconds
[2024-11-06T20:17:06.441+0000] {processor.py:186} INFO - Started process (PID=5776) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:17:06.442+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:17:06.442+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:17:06.442+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:17:06.460+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:17:06.484+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:17:06.483+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:17:06.507+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:17:06.507+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:17:06.521+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.084 seconds
[2024-11-06T20:17:37.019+0000] {processor.py:186} INFO - Started process (PID=5833) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:17:37.020+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:17:37.021+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:17:37.021+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:17:37.043+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:17:37.062+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:17:37.061+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:17:37.091+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:17:37.091+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:17:37.124+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.111 seconds
[2024-11-06T20:18:07.377+0000] {processor.py:186} INFO - Started process (PID=5892) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:18:07.380+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:18:07.382+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:18:07.382+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:18:07.402+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:18:07.443+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:18:07.443+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:18:07.467+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:18:07.467+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:18:07.483+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.111 seconds
[2024-11-06T20:18:09.043+0000] {processor.py:186} INFO - Started process (PID=5922) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:18:09.044+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:18:09.045+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:18:09.045+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:18:09.068+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:18:09.182+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:18:09.182+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:18:09.199+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:18:09.199+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:18:09.217+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.178 seconds
[2024-11-06T20:18:44.481+0000] {processor.py:186} INFO - Started process (PID=5960) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:18:44.483+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:18:44.485+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:18:44.484+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:18:44.521+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:18:44.791+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:18:44.787+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:18:44.858+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:18:44.858+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:18:44.889+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.416 seconds
[2024-11-06T20:19:54.564+0000] {processor.py:186} INFO - Started process (PID=6006) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:19:54.568+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:19:54.571+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:19:54.570+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:19:54.612+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:20:16.889+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:20:16.885+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:20:16.934+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:20:16.933+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:20:16.955+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 21.140 seconds
[2024-11-06T20:20:47.625+0000] {processor.py:186} INFO - Started process (PID=6069) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:20:47.625+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:20:47.627+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:20:47.626+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:20:47.642+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:20:47.659+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:20:47.658+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:20:47.677+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:20:47.677+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:20:47.691+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.071 seconds
[2024-11-06T20:21:18.888+0000] {processor.py:186} INFO - Started process (PID=6132) to work on /opt/airflow/dags/user_processing.py
[2024-11-06T20:21:18.889+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/user_processing.py for tasks to queue
[2024-11-06T20:21:18.890+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:21:18.890+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/user_processing.py
[2024-11-06T20:21:18.909+0000] {processor.py:925} INFO - DAG(s) 'user_processing' retrieved from /opt/airflow/dags/user_processing.py
[2024-11-06T20:21:18.961+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:21:18.961+0000] {dag.py:3239} INFO - Sync 1 DAGs
[2024-11-06T20:21:18.984+0000] {logging_mixin.py:190} INFO - [2024-11-06T20:21:18.984+0000] {dag.py:4180} INFO - Setting next_dagrun for user_processing to 2024-11-05 00:00:00+00:00, run_after=2024-11-06 00:00:00+00:00
[2024-11-06T20:21:18.999+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/user_processing.py took 0.120 seconds
